Assets/Python/Afforess/DiplomacySettings.py

```
@@ -1,108 +1,260 @@
-	GC.setDefineINT("CAN_TRADE_PEACE", ANewDawnOpt.isCanTradePeace())
-	GC.setDefineINT("CAN_TRADE_WAR", ANewDawnOpt.isCanTradeWar())
-	GC.setDefineINT("CAN_TRADE_EMBARGO", ANewDawnOpt.isCanTradeEmbargo())
-	GC.setDefineINT("CAN_TRADE_CIVICS", ANewDawnOpt.isCanTradeCivics())
-	GC.setDefineINT("CAN_TRADE_RELIGIONS", ANewDawnOpt.isCanTradeReligions())
-	GC.setDefineINT("CAN_TRADE_OPEN_BORDERS", ANewDawnOpt.isCanTradeOpenBorders())
-	GC.setDefineINT("CAN_TRADE_LIMITED_BORDERS", ANewDawnOpt.isCanTradeLimitedBorders())
-	GC.setDefineINT("CAN_TRADE_DEFENSIVE_PACT", ANewDawnOpt.isCanTradeDefensivePact())
-	ANewDawnOpt.setCanTradeAlliance(GAME.isOption(GameOptionTypes.GAMEOPTION_ENABLE_PERMANENT_ALLIANCES))
-	ANewDawnOpt.setAdvancedDiplomacy(GAME.isOption(GameOptionTypes.GAMEOPTION_ADVANCED_DIPLOMACY))
+    """Optimized initialization using pre-cached methods and batch operations"""
+    print
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)